### DISCUSSION TITLE
Enabling Set Withdraw Address Transaction Feature

### DISCUSSION THREAD

**Discussion by MeriaIT** [fetchai/FIPs#23] [Ideas]
Hello everyone,
We launched our validator recently and during the configuration we noticed that it was not possible to set a Withdraw address due to `WithdrawAddrEnabled` being `false`. Before submitting the proposal on-chain, we wanted to discuss with you about the idea of setting the value to `true` and allow validator to segregate the withdraw address from the validator wallet. 

Below is the proposal we'd like to submit :

```
{
  "title": "Enabling Set Withdraw Address Transaction Feature",
  "description": "This proposal advocates for the authorization of the 'set withdraw address' transaction. The plan necessitates modifying the chains parameter to set WithdrawAddrEnabled as True. This adjustment allows validator to get their rewards on another wallet (specified by the set-withdrawal-addr transaction) instead of the wallet which created the validator. If no withdrawal address is set, rewards will go on the wallet that created the validator. Presently, this functionality is non-operational on FetchAI. Enabling it would significantly streamline and transform validator's regular operations.",
  "changes":[
        {
         "subspace":"distribution",
         "key":"WithdrawAddrEnabled",
         "value":"true"
      }
      ],
  "deposit": "2048000000000000000000afet"
}
```

We're looking forward discussing with you about this proposal !
Meria's Node operation team :)

**Comment by DDeAlmeida** [comment]
Hi, yeah good point.
Thank you

**Comment by Jackson-DKMG** [comment]
Great idea.
Been meaning to use that before and was a bit frustrated when I realised it's not possible.

**Comment by qf3l3k** [comment]
I will be happy to vote for this feature to be enabled. It will definitely simplify rewards collection.

**Comment by packetstracer** [comment]
We think it's a nice feature, specially in terms of node operators security, already available in other chains.

Will vote Yes.

**Comment by cosmosscanner** [comment]
Excellent proposal! It would allow us to autostake rewards and commissions easier, Restake.app does not support Ledger.

**Comment by antrixy** [comment]
I am yes. Its a no brainer.

**Comment by ivandiazperez** [comment]
Glad to see this coming to cosmos chains as it is already avb in other chains since some time ago

**Comment by paddyson79** [comment]
you've got my support for this, nice feature
